hack/local/run-tests.sh

```diff
@@ -1,9 +1,13 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
 #
 # SPDX-License-Identifier: Apache-2.0
 
+set -o errexit
```

Comment:
That is because the test was failing very late at https://github.com/gardener/oidc-webhook-authenticator/blob/7f88563f79448c7d8ca70944b36487746b301b45/test/integration/integration_suite_test.go#L88 with error that the binary cannot be found, and that was because the `go build ...` below were failing but script did not exit early with an error.